alerts.json

```diff
@@ -4,9 +4,14 @@
         "message": "",
         "comment": "Inactive alerts will not be shown. You can use markdown, like _this_, **this**, or [this](https://pokeapi.co). Possible levels are 'info', 'important' and 'success'. The default is 'info'. Important alerts are sticky and stay on-screen until dismissed by the user."
     },
+    {
+        "level": "success",
+        "active": true,
+        "message": "Data from last year's **Sword and Shield** games (generation 8) is currently **being added** to PokéAPI. While waiting for an upstream project ([Veekun](https://github.com/veekun/pokedex)) for official data we decided to allow our community to bring in new data from various different sources. If you want to partecipate to this update project [read more on GitHub](https://github.com/PokeAPI/pokeapi/issues/520)."
```

Comment:
I think we should say "verified data" rather than "official data". Official implies that Nintendo supplied the data directly, when that is not the case.

Also, participate is misspelt.